go/src/meguca/templates/body.go

```diff
@@ -198,6 +216,7 @@ func (c *bodyContext) formatCode(frag string, fn func(string)) {
 
 // Inject spoiler tags and call fn on the remaining parts
 func (c *bodyContext) parseSpoilers(frag string, fn func(string)) {
+	c.state.rbText = config.GetBoardConfigs(c.board).RbText
```

Comment:
Please don't sync to mutex on each spoiler parse. Do this once per render batch. Preferably in ``thread.qtpl:renderThreadPosts``, etc.

Reply:
I got the others just fine, but I'm a bit confused about this one.
I'm having a hard time getting the state of rbText in `thread.qtpl:renderThreadPosts` to `templates/body.go`.
I tried various setters and getters but I can't seem to get what I want from `body.go` into ``thread.qtpl:renderThreadPosts` or vice-versa.

Reply:
Do you rerun `make server` after qtpl file changes?

Reply:
Do you rerun `make server` after qtpl file changes?